docs/openapi/schemas.yaml

```diff
@@ -40,6 +40,9 @@ Domain:
 ImsOrganizationId:
   type: string
   example: '1234567890ABCDEF12345678@AdobeOrg'
+ImsUserId:
+  type: string
+  example: '8G5129DA63CE4CD98A495EAD@8053114e63a16b59485e3c.e'
```

Comment:
This looks like a legit id.  Maybe use than obviously fake pattern to show an example: `123abc456def789fgh@09876543e12a345e6.e`

Reply:
Good point. I took a real one and modified the letters and alphabets.